easybuild/easyconfigs/t/TRIQS-cthyb/TRIQS-cthyb-3.1.0-foss-2022a.eb

```
@@ -0,0 +1,69 @@
+easyblock = 'CMakeMake'
+
+name = 'TRIQS-cthyb'
+version = '3.1.0'
+
+homepage = 'https://triqs.github.io/cthyb/'
+description = """
+ TRIQS (Toolbox for Research on Interacting Quantum Systems) is a
+ scientific project providing a set of C++ and Python libraries to
+ develop new tools for the study of interacting quantum systems.
+
+ cthyb = continuous-time hybridisation-expansion quantum Monte Carlo
+
+ The TRIQS-based hybridization-expansion solver allows to solve the
+ generic problem of a quantum impurity embedded in a conduction bath
+ for an arbitrary local interaction vertex. The “impurity” can be any
+ set of orbitals, on one or several atoms.
+"""
+
+docurls = ['https://triqs.github.io/cthyb/%(version_major_minor)s.x/']
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/TRIQS/cthyb/releases/download/%(version)s/']
+sources = ['cthyb-%(version)s.tar.gz']
+checksums = ['8d6d2c4d5b3928d062b72fad4ea9df9aae198e39dd9c1fd3cc5dc34a5019acc0']
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2022.05'),
+    ('Boost', '1.79.0'),
+    ('Clang', '13.0.1'),
+    ('Clang-Python-bindings', '13.0.1'),
+    ('GMP', '6.2.1'),
+    ('HDF5', '1.13.1'),
```

odd versions of HDF5 are development versions, stick to the stable even versions:
```suggestion
    ('HDF5', '1.12.2'),
```